nbextensions/advancedViewCell/widgets/appParamsViewWidget.js

```diff
@@ -503,147 +460,129 @@ define([
                 // based on the param ordering (layout), render the html layout,
                 // with an id mapped per parameter in this set
 
-                return (
-                    Promise.resolve()
-                        .then(() => {
-                            if (inputParams.layout.length === 0) {
-                                places.inputFields.innerHTML = span(
-                                    { style: { fontStyle: 'italic' } },
-                                    'This app does not have input objects'
-                                );
-                            } else {
-                                places.inputFields.innerHTML = inputParams.content;
-                                return Promise.all(
-                                    inputParams.layout.map((parameterId) => {
-                                        const spec = inputParams.paramMap[parameterId];
-                                        try {
-                                            return makeFieldWidget(
-                                                appSpec,
-                                                spec,
-                                                model.getItem(['params', spec.id])
-                                            ).then((widget) => {
-                                                widgets.push(widget);
-
-                                                return widget.start({
-                                                    node: document.getElementById(
-                                                        inputParams.view[parameterId].id
-                                                    ),
-                                                });
+                return Promise.resolve()
```

Comment:
remove unnecessary bracket, which decreases the indentation of this whole section by one tab,